etc/requirements_locked.txt

```diff
@@ -35,7 +35,7 @@ flask-cors==2.1.3
 flask==1.1.1              # via flask-cors
 funcsigs==1.0.2           # via mock, python-interface
 futures==3.2.0            # via tornado
-h5py==2.7.1
+h5py==2.10.0
```

Comment:
This update was necessary to read the file's attr in the test...`2.7.1` was failing with a strange numpy c error.